configs/tasks/rearrangepick_replica_cad_example.yaml

```diff
@@ -87,14 +92,29 @@ TASK:
             TYPE: "ArmAction"
             ARM_CONTROLLER: "ArmVelAction"
             GRIP_CONTROLLER: "MagicGraspAction"
-            GRASP_THRESH_DIST: 0.15
+            GRASP_THRESH_DIST: 5.15
             DISABLE_GRIP: False
             VEL_CTRL_LIM: 0.0125
+        BASE_VEL:
+            TYPE: "BaseVelAction"
+            LIN_SPEED: 12.0
+            ANG_SPEED: 12.0
+            ALLOW_DYN_SLIDE: True
+            END_ON_STOP: False
+            ALLOW_BACK: True
+            MIN_ABS_LIN_SPEED: 1.0
+            MIN_ABS_ANG_SPEED: 1.0
+        EMPTY:
```

Comment:
Why do we need empty action?

Reply:
Some policies need it and it's good for interactive play.